about.html

```diff
@@ -8,57 +8,92 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Hind+Siliguri&display=swap" rel="stylesheet">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
+  <div class="container">
+   
+    <section class="top">
+      <header>
+
+          
+          <div class="Logo">
+              <img src="/img/lambda-black.png" alt="Logo"> 
+           </div>
+          
+           <div class="navbar">
+           <a href="/index.html">Home</a>
```

Comment:
Great Job linking to both index.html and about.html. This will prevent the user of the website from getting stuck while navigating, which will provide a good user experience